ultimate-pipeline/conf/base/catalog.yml

```diff
@@ -60,6 +60,47 @@ yolo_detect_annotation_val:
     type: pandas.CSVDataset  # shorthand notation for the dataset which will handle individual partitions
     save_args:
       columns: ["cls","x","y","w","h"]
+      header: False
+      index: False
+      sep: " "
+      decimal: .
+
+# YOLO Pose/Keypoints Estimation
+# The following datasets are used to train a YOLO model for pose estimation
+
+yolo_keypoints_images_train:
```

Comment:
To consider: Use Kedro [dataset factories](https://docs.kedro.org/en/stable/data/kedro_dataset_factories.html) to simplify/parameterize the catalogue

Reply:
Implemented with dataset factories - `catalog.yml` is much shorter now.